src/components/Background.vue

```diff
@@ -1,17 +1,29 @@
 <template>
-  <div id="background" class="grid-background">
-    <slot/>
+  <div
+    id="background"
+    class="grid-background"
+    :style="{backgroundPosition: `${-scroll.x}px ${-scroll.y}px`}"
```

Comment:
Inserting spaces to seperate the object curly braces might be more consistent in terms of codestyle. Have you done it this way to be more compact?